test/parallel/test-whatwg-url-properties.js

```diff
@@ -25,6 +25,9 @@ const expected = ['toString',
                   'username', 'password', 'host', 'hostname', 'port',
                   'pathname', 'search', 'searchParams', 'hash', 'toJSON'];
 
+const obj = { toString() { throw new Error('toString'); } };
```

Comment:
Right, I overlooked those other tests that have Node.js-specific ones following WPT. Should now be addressed in 786aead7de3898012512915a10f4685a13f478d0